src/main/kotlin/no/nav/syfo/integration/oauth2/OAuth2Components.kt

```diff
@@ -11,16 +11,16 @@ import no.nav.security.token.support.client.core.http.OAuth2HttpClient
 import no.nav.security.token.support.client.core.http.OAuth2HttpRequest
 import no.nav.security.token.support.client.core.oauth2.OAuth2AccessTokenResponse
 import no.nav.security.token.support.ktor.TokenValidationContextPrincipal
-import org.slf4j.LoggerFactory
+import no.nav.syfo.util.logger
 import java.util.Optional
 
 class DefaultOAuth2HttpClient(private val httpClient: HttpClient) : OAuth2HttpClient {
-    val logger = LoggerFactory.getLogger(this::class.java)
+    private val logger = this.logger()
 
     override fun post(oAuth2HttpRequest: OAuth2HttpRequest): OAuth2AccessTokenResponse {
         return runBlocking {
             try {
-                return@runBlocking httpClient.submitForm(
+                httpClient.submitForm(
```

Comment:
`runBlocking` er en lambda, så `return` trengs ikke, den returnerer det som står på siste linje uansett.

Reply:
Lurer på om ikke intellij tvang oss til å sette den return@runBlocking tidligere. Vet du noe om det?

Reply:
Jeg gjetter at koden eksisterte uten `runBlocking` til å begynne med, og når det ble lagt til så forslo editoren å legge til `@runBlocking` for å fikse returnen, selv om man heller bare burde fjernet den.